### PR TITLE
#5 Task.AlexandrKolyadich

### PR DESCRIPTION
**Pull request by AlexandrKolyadich**
https://travis-ci.org/github/AlexandrKolyadich/js-assignments/builds/709243244

### CODE REVIEW

File: task/05-regex-tasks.js [hunk 1]
```diff
@@ -31,7 +31,9 @@
  * @return {RegExp}
  */
 function getRegexForGuid() {
-   throw new Error('Not implemented');
+   return /{[\d, a-f, A-F]{8}-[\d, a-f, A-F]{4}-[\d, a-f, A-F]{4}-[\d, a-f, A-F]{4}-[\d, a-f, A-F]{12}}/;
+
+   //throw new Error('Not implemented');
 }
```
Comment (FedorovD) [inline task/05-regex-tasks.js:37]: three times repeating part [\d, a-f, A-F]{4}- could be reduced to ([\d, a-f, A-F]{4}-){3}
unnecessary `,` and ` `  signs

File: task/05-regex-tasks.js [hunk 2]
```diff
@@ -91,7 +97,9 @@ function getRegexForIPv4() {
  * @return {RegExp}
  */
 function getRegexForSSN() {
-   throw new Error('Not implemented');
+   return /^([1-9][0-9][0-9]|[0-9][1-9][0-9]|[0-9][0-9][1-9])-([1-9][0-9]|[0-9][1-9])-([1-9][0-9][0-9][0-9]|[0-9][1-9][0-9][0-9]|[0-9][0-9][1-9][0-9]|[0-9][0-9][0-9][1-9])$/;
+
```
Comment (FedorovD) [inline task/05-regex-tasks.js:101]: could be easier and shorter, try another way

**Comment by FedorovD** [comment]
I'll mark the PR as "Draft", please click "ready for review" when it will be finished. Thank you!

**Comment by AlexandrKolyadich** [comment]
fixed

**Comment by FedorovD** [comment]
Closing the PR as accepted! Thanks!

**Pull request closed without merging**